### PR TITLE
Adding ROSXBee to documentation index for foxy

### PR DESCRIPTION
**Pull request by Sudharsan10**
I'd like ROSXBee to be indexed and documented on the ros.org

### CODE REVIEW

File: foxy/distribution.yaml [hunk 1]
```diff
@@ -3612,6 +3612,11 @@ repositories:
       url: https://github.com/ros2/rosidl_typesupport_gurumdds.git
       version: foxy
     status: developed
+  rosxbee:
+    doc:
```
Comment (ivanpauno) [inline foxy/distribution.yaml:3616]: could you add a `source` entry?

**Review by ivanpauno — CHANGES_REQUESTED** [review]
the package.xml must have the license entry filled https://github.com/Sudharsan10/ROSXBee/blob/65c24381f598f6bcdf41587f0fdf488138bd96dd/code/src/rosxbeepy/package.xml#L9.

**Comment by ivanpauno** [comment]
do you need both PRs or can this be closed in favor of https://github.com/ros/rosdistro/pull/28597/?

File: foxy/distribution.yaml (hunk 1 above)
Comment (Sudharsan10) [reply foxy/distribution.yaml:3616]: Thanks @ivanpauno I will make an entry and will update here.
Also will fill the license information in the package.

**Comment by Sudharsan10** [comment]
> do you need both PRs or can this be closed in favor of #28597?

yes you can close this one. Also would love to know why my pull request failed in the automated test so that I can learn resubmit my pull request again for the #28597.

File: foxy/distribution.yaml (hunk 1 above)
Comment (Sudharsan10) [reply foxy/distribution.yaml:3616]: @ivanpauno I have made both the changes, Let me know if I should anything else. I'm new to this so dont mind if I ask dumb questions

**Pull request closed without merging**